### PR TITLE
Added VLC and f.lux

### PR DESCRIPTION
**Pull request by arijitAD**

### CODE REVIEW

File: install/flux.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+# f.lux install steps
+
+DESC='--- f.lux successfully installed ---'
+
+# __BEGIN_MAIN__
+sudo add-apt-repository ppa:nathan-renniewaldock/flux
+sudo apt-get update
+sudo apt-get install fluxgui
+# __END_MAIN__
```
Comment (andreybleme) [inline install/flux.sh]: Where are you showing the success message?

Missing "echo" @arijitAD

**Comment by arijitAD** [comment]
Added Echo.

**Comment by andreybleme** [comment]
@arijitAD That's fine now! :+1: 
Thanks for helping.

**Comment by arijitAD** [comment]
Happy to help I will add more scripts tomorrow.

**Pull request merged**